setup.cfg

```diff
@@ -0,0 +1,57 @@
+[metadata]
+name = keylime
+version = 6.3.1
```

Comment:
Is there any way to have this templated off of the actual release number in github? Not show stopping just would be nice to never have to worry about updating.